docs/content/docs/recipes/task-manager-diceroller.md

```diff
@@ -0,0 +1,384 @@
+    const getInitialObjects = async () => {
+      // TODO 1: Configure the container.
+      // TODO 2: Get the container from the Fluid service.
+      // TODO 3: Return the Fluid SharedMap and TaskManager DDS objects.
+    };
+    ```
+
+1. Replace `TODO 1` with the following code.
+
+    ```ts
+    const client = new TinyliciousClient();
+    const containerSchema: ContainerSchema = {
+      initialObjects: {
+        sharedMap: SharedMap,
+        taskManager: TaskManager,
+      },
+    };
+    ```
+
+1. Replace `TODO 2` with the following code. Note that `containerId` is being stored on the URL hash, and if there is no `containerId` a new container is created instead.
+
+    ```ts
+    let container: IFluidContainer;
+    const containerId = window.location.hash.substring(1);
+    if (!containerId) {
+      ({ container } = await client.createContainer(containerSchema));
+      const id = await container.attach();
+      window.location.hash = id;
+    } else {
+      ({ container } = await client.getContainer(containerId, containerSchema));
+    }
+    ```
+
+1. Replace `TODO 3` with the following code.
+
+    ```ts
+    return {
+      sharedMap: container.initialObjects.sharedMap as ISharedMap,
+      taskManager: container.initialObjects.taskManager as ITaskManager,
+    };
+    ```
+
+### Get the Fluid data on application startup
+
+Now that you have defined how to get our Fluid data, you need to tell React to call `getInitialObjects()` when the application starts up and then store the result in state. Add the following at the top of the `App()` function.
+
+  ```ts
+  // TODO 1: Setup const values
+  // TODO 2: Setup state
+  // TODO 3: Get Fluid data on app startup
+  ```
+
+1. Replace `TODO 1` with the following code. These values will be used when interacting with our DDSes.
+
+    ```ts
+    const diceKey = "dice-key";
+    const diceRollTaskId = "diceRollTaskId";
+    ```
+
+2. Replace `TODO 2` with the following code. Note about this code:
+   - These are the state variables that will be used by the React app.
+   - Some of these values are only used in the debug information display. Feel free to remove any that are not applicable in your applicaiton.
+
+    ```ts
+    const [sharedMap, setSharedMap] = React.useState<ISharedMap>();
+    const [taskManager, setTaskManager] = React.useState<ITaskManager>();
+    const [diceValue, setDiceValue] = React.useState<number>();
+    const [assigned, setAssigned] = React.useState<boolean>(false);
+    const [queued, setQueued] = React.useState<boolean>(false);
+    const [subscribed, setSubscribed] = React.useState<boolean>(false);
+    ```
+
+3. Finally, replace `TODO 3` with the following code. Note about this code:
+    - Passing an empty dependency array as the last parameter of `useEffect` ensures that this function is called only once.
+    - Since `setSharedMap` and `setTaskManager` are state-changing methods, it will cause the React `App` component to immediately rerender.
```

Comment:
```suggestion
    - Since `setSharedMap` and `setTaskManager` are state-changing methods, they will cause the React `App` component to immediately rerender.
```